Tests/DownloadServiceTests.swift

```diff
@@ -16,27 +16,29 @@ private let VersionFileURL = "https://example.com/versions.json"
 class DownloadServiceTests: XCTestCase {
   let service = DownloadService(string: VersionFileURL)
 
-  override func setUp() {
-    LSNocilla.sharedInstance().start()
-  }
-
   override func tearDown() {
-    LSNocilla.sharedInstance().stop()
+    OHHTTPStubs.removeAllStubs()
+    super.tearDown()
   }
 
   func testFetchWhenResponseIsValid() {
-    stubRequest("GET", VersionFileURL)
-      .andReturn(200)
-      .withBody(String(data: fixtureData("Versions"), encoding: NSUTF8StringEncoding))
-
+    stub(condition: isHost("example.com")) { _ in
```

Comment:
So i couldn't figure out how to get two matchers in OHTTPStubs to work nicely, but i also kind of didn't care -- it's not like we have to make a distinction between two types of requests here since there is only one.